package.json

```diff
@@ -1,19 +1,22 @@
 {
-  "name": "@maarkllc/eslint-config-default",
+  "name": "@maarkllc/linter",
```

Comment:
Are these names in this file obsolete btw?

Reply:
It's obsolete, already fixed